src/main/java/uk/gov/ch/model/officer/active/ActiveOfficerDetails.java

```diff
@@ -221,6 +229,19 @@ public void setDateOfAppointment(String dateOfAppointment) {
         this.dateOfAppointment = dateOfAppointment;
     }
 
+    public void setCorporate(boolean corporate) {
+        this.corporate = corporate;
+    }
+
+    public void setAppointmentType(String appointmentType) {
+        this.appointmentType = appointmentType;
+    }
+
+    public String getOfficerType() {
+        StringBuilder ty = new StringBuilder(corporate ? "corporate " : "natural ");
```

Comment:
i'd move the building of this string to the cs-api, as other teams might want the raw data from this class, and the concatenation of eg "natural" and appointment type is probably only meaningful to the CS service. getOfficerType should return the private String officerType;

Reply:
sounds like the officerType field is not used if getOfficerType generates and returns a different string. Delete 
@Transient
    @JsonProperty("officer_type")
    private String officerType;

and leave the getOfficerType(0 in there

Reply:
I'm not sure what the ty variable means, can you rename to officerType